packages/dataclass-editor/src/translation/dataclass-editor/ja.json

```diff
@@ -0,0 +1,105 @@
+{
+  "browser": {
+    "dataClasses": "",
+    "ivyScriptTypes": "",
+    "ownTypes": "",
+    "searchAllTypes": "",
+    "typeAsList": ""
+  },
```

Comment:
don't merge me :warning: ... this is just a template to be filled by JP translators.

Reply:
fyi @ivy-jal , if Weblate creates new languages the values are created empty, so if the dev's merge that stuff we get these UIs without any string. 
I see no possibility to disable that behavior.
![weblate-disable-empty-creation](https://github.com/user-attachments/assets/fbe64767-b479-426f-a3ba-c1f105d65f4e)



However, I think we're not hurt by this, devs simply need to know that such translations should not be merged, until the real contents are added. I'll raise the awareness for this in the Daily tomorrow.

Reply:
fyi @ivy-edp @ivy-lmu @ivy-lli 

If you get weblate PRs with empty JSON files, do not merge it yet, or all labels are gone if you switch to this language. If you like you may close such branches until translators file real contents. They will be auto re-created after 7 days however.